src/hooks/useRecipe.js

```diff
@@ -19,14 +19,14 @@ export default function useRecipe({ initialRecipeData }) {
     const ingredientsArray = ingredients.split(',').map(item => item.trim())
 
     const addRecipe = [
-      ...recipeData,
       {
         name: recipeName,
         instructions: instructions,
         ingredients: ingredientsArray,
         image: imgUrl,
         id: id,
```

Comment:
Just a tip - it is also fine as you are doing it: You can use shorthand properties, if the key of the property matches the variable name you're setting it to exactly:

```suggestion
        name: recipeName,
        instructions,
        ingredients: ingredientsArray,
        image: imgUrl,
        id,
```

I often find myself explicitly naming variables in a way that enables me to "shorthand" them into an object. ;)

See here: https://ui.dev/shorthand-properties/

Reply:
Thanks, I did not know about that!